includes/helpers/class-accredible-learndash-admin-table-helper.php

```diff
@@ -430,14 +431,16 @@ class="button accredible-button-outline-natural accredible-button-small"
 		/**
 		 * Resolves href attribute for pagination.
 		 *
+		 * @param string $page page.
+		 *
 		 * @param string $page_num page num used passed to href.
 		 *
 		 * @return string
 		 */
-		private static function get_pagination_href( $page_num ) {
+		private static function get_pagination_href( $page, $page_num ) {
```

Comment:
I think the order of these parameters are incorrect, when you're calling the method, the `page` is the second parameter

Reply:
I have updated the calls to this function to have this order.